### DISCUSSION TITLE
Call for testing Freeplane 1.11.1_08 alpha

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#960] [Announcements]
The latest preview considers many of your suggestions, and I would like to open it for translations and to remove the alpha label. It can be done only if no data format changes is needed any longer. Please test it and give me your feedback before next Friday, January 27th. 

Regards,
Dimitry

**Comment by abc16361** [comment]
The node shape Bubble has been changed to a new one with a bigger round corner since the recent fp-1.11.1-05 version. I like the bubble shape on fp-1.10.xx and use it to store small pieces of data. It is compact and looks delicate. If the new bubble shape is planned to be adopted, please keep the old one and name it as ' Round rectangle '. Thanks

![bubble](https://user-images.githubusercontent.com/101362122/213873109-62a45288-b4b3-4a08-9696-65dbd6e955db.png)

PS.  I made a request on the [Issues](https://github.com/freeplane/freeplane/issues/961).

**Reply by sifran-github** [reply]
I didn't check the preview yet, but i agree that the original bubbles should remain. I use it all the time.

**Reply by dpolivaev** [reply]
fixed

**Comment by macmarrum** [comment]
1.11.x is missing layout options from 1.10.x: _first child with parent, last child with parent, as parent__, as mentioned in
- https://github.com/freeplane/freeplane/discussions/892#discussioncomment-4549062

I've come to rely on these layouts, and therefore it's doubtful that I'll upgrade from 1.10.x.

**Reply by dpolivaev** [reply]
restored

**Comment by Lotus49b** [comment]
I'm working on this layout (Kanban-ish). 
![image](https://user-images.githubusercontent.com/11365744/214027676-03c3fc08-a102-4886-947c-cffe8464c339.png)
Problem 1: On Level 2 I've selected the "top to bottom, align right, align child nodes after parent" but the edge line is running through the nodes, not next to them on the left (as per indicator arrow 1).
Problem 2: When saving the node layout style style for level 2 (and others) the change is applied to some level 2 nodes but not all (as per indicator arrow 3).
Problem 3: When changing "Base distance to children" and "Distance between children", the change for one of those is reset to default when saving. I cannnot get both parameters to save, only one. (indicator 2)

I'm not seeing errors in the logs.

I'm working with version 1.11.1_05 now, as that allows me to get the edges on the left of the lower nodes, but the other issues I mentioned exist there too.
P.S. Unrelated (I think) - it would be nice if I could get the connector kines (seen in orange) to connect to the left side of the right-hand list.

**Reply by dpolivaev** [reply]
Please submit an issue and attach a mind map so that I can easier reproduce your findings

**Reply by Lotus49b** [reply]
Will do.

**Reply by dpolivaev** [reply]
problem 1 fixed

**Comment by sifran-github** [comment]
1. All these icons take a lot of screen space. I would prefer to have a combo box, and only when clicked to choose the layout.

2. Alternatively, i'm thinking about a different way to choose the layout. it is possible to replace all the icons with the following combo boxes:
A. CHILDREN_LOCATION: up, down, left, right...
B. CHILDREN_ORDER: up->right, left->right, right->left ... etc
C. AUTO: checkbox yes/no (?)
so just 3 small combo boxes to include all the options?
I'm actually not sure that it is better. but it takes less space on the screen.

3. the keyboard is still messy like i described here: https://github.com/freeplane/freeplane/discussions/892#discussioncomment-4570626

**Reply by dpolivaev** [reply]
I changed the implementation to occupy less space on the panel

**Reply by sifran-github** [reply]
@dpolivaev nice, I will check. what about the keyboard issue? I think that its important because it breaks down the usability of the keyboard. it should be addressed IMHO. I wonder what do you think about my proposed solution here : https://github.com/freeplane/freeplane/discussions/892#discussioncomment-4570626 which will let the user choose their keyboard layout based on function (my preferred choice) or based on the direction.

**Comment by abc16361** [comment]
Layout patterns on layout box do not change accordingly (freeplane-1.11.1-alpha-pre09u1)

After setting different layouts on different node branches, layout patterns on the layout box do not change accordingly if node branches are clicked one by one. It always shows the last set layout. Further, open the box, the layout pattern always stay on the last set pattern. When freeplane is activated and a map is open, it is expected that the layout pattern set on the node selected is shown, but the Node layout just displays the first one.  
![LAYOUT_ed](https://user-images.githubusercontent.com/101362122/215528265-3e40edb1-0f01-4986-8842-ccc458ddb608.png)

PS. alse reported on the [Issues](https://github.com/freeplane/freeplane/issues/971)

**Comment by sifran-github** [comment]
![image](https://user-images.githubusercontent.com/5730355/217260821-3713dd4a-7893-4dcb-9b7c-cc9f95c73b8c.png)

**Reply by dpolivaev** [reply]
Please always attach mind maps and explicitly write which preview version you use

**Reply by sifran-github** [reply]
mindmap: the mindmap from this comment. https://github.com/freeplane/freeplane/discussions/892#discussioncomment-4570531
version: freeplane-1.11.1-alpha-pre08

**Reply by dpolivaev** [reply]
I think it was fixed. Please check the latest preview.

**Reply by sifran-github** [reply]
correct. it is fixed in freeplane-1.11.1-pre12.
didn't see the new version.

**Comment by EdoFro** [comment]
Dear @dpolivaev,
I didn't want to take part of the discussion about the new layouts functionality, because I don't know If I'm going to use it, because I haven't need it until this moment. 
I wanted not to influence the discussion and let the interested ones decide about the feature.

I have allways thought that all different layouts should be available as different views for the same map, and no specific view information should be part of the nodes in the mindmap file. For example, jump into a node and then 'turning on' the "organigram view" to see it ordered top to bottom. Or jump to another one and turn on the "ishikawa" view, or the "kanban" view, etc.

But I can understand that there are users that need this change and it's ok.

There are, although, some points I want to ask for:
1. If a map doesn't use the new layouts, then the mindmap file should be compatible with previous Freeplane versions.
   Last time I tested; if a user opens a FP1.10 mindMap with the new version and edits a text and then saves. FP1.10 can't open it any more.

2. When using the new layouts in a Map, I would like to have the possibilty to open the map in a "right left only" layout that ignores all the top/down node's information, to work in the map as always, using my standard hotkeys and focusing on productivity.
   Then, when I want to show the map to others, I select the MindMap view and I take care about layout.

Jm2c,

edo
